plugins/modules/utm_aaa_group.py

```diff
@@ -8,120 +8,117 @@
-        type: str
-        default: none
-        choices:
-          - none
-          - ipsec_dn
-          - directory_groups
-    edirectory_groups:
-        description:
-          - List of edirectory group strings.
-        type: list
-        elements: str
-        default: []
-    ipsec_dn:
-        description:
-          - The ipsec dn string.
-        type: str
-        default: ''
-    ldap_attribute:
-        description:
-          - The ldap attribute to check against.
-        type: str
-        default: ''
-    ldap_attribute_value:
-        description:
-          - The ldap attribute value to check against.
-        type: str
-        default: ''
-    members:
-        description:
-          - A list of user ref names (aaa/user).
-        type: list
-        elements: str
-        default: []
-    network:
-        description:
-          - The network reference name. The objects contains the known ip addresses for the authentication object (network/aaa).
-        type: str
-        default: ""
-    radius_groups:
-        description:
-          - A list of radius group strings.
-        type: list
-        elements: str
-        default: []
-    tacacs_groups:
-        description:
-          - A list of tacacs group strings.
-        type: list
-        elements: str
-        default: []
+  name:
+    description:
+      - The name of the object. Will be used to identify the entry.
+    type: str
+    required: true
+  adirectory_groups:
+    description:
+      - List of adirectory group strings.
+    type: list
+    elements: str
+    default: []
+  adirectory_groups_sids:
+    description:
+      - Dictionary of group sids.
+    type: dict
+    default: {}
+  backend_match:
+    description:
+      - The backend for the group.
+    type: str
+    choices:
+      - none
+      - adirectory
+      - edirectory
+      - radius
+      - tacacs
+      - ldap
+    default: none
+  comment:
+    description:
+      - Comment that describes the AAA group.
+    type: str
+    default: ''
+  dynamic:
+    description:
+      - Group type. Is static if none is selected.
+    type: str
+    default: none
+    choices:
+      - none
+      - ipsec_dn
+      - directory_groups
+  edirectory_groups:
+    description:
+      - List of edirectory group strings.
+    type: list
+    elements: str
+    default: []
+  ipsec_dn:
+    description:
+      - The ipsec dn string.
+    type: str
+    default: ''
+  ldap_attribute:
+    description:
+      - The ldap attribute to check against.
+    type: str
+    default: ''
+  ldap_attribute_value:
+    description:
+      - The ldap attribute value to check against.
+    type: str
+    default: ''
+  members:
+    description:
+      - A list of user ref names (aaa/user).
+    type: list
+    elements: str
+    default: []
+  network:
+    description:
+      - The network reference name. The objects contains the known ip addresses for the authentication object (network/aaa).
```

Comment:
```suggestion
      - The network reference name. The objects contains the known IP addresses for the authentication object (network/aaa).
```

Reply:
There are many improvements to be made in those docs, but manually.

Reply:
True :) I'm adding suggestions for ones that I notice while skimming over the changes, so we can already fix some of them now.